src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -19,6 +19,11 @@
       <% } %>
       <a href="/about.jsp">About</a>
       <a href="/activityfeed">Activity Feed</a>
+      <% if (request.getSession().getAttribute("user") != null) { %>
+        <a href="/profile/<%=request.getSession().getAttribute("user")%>">Profile</a>
+      <% } else{ %>
```

Comment:
No need to have an else here at all--you can just have the if part and do nothing otherwise (instead of putting an empty `a` tag that doesn't do anything).